mbstrdecoder/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mbstrdecoder" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9df7778c0c051c98f6b5a848909202e5cd423c9692665b20d1a9bd0f771b8235
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - releasecmd
```

Comment:
Is this really needed? The recipe was able to pass without this being available in conda-forge...

Reply:
The recipe was in the incorrect location... That's why it passed.